saltychat/SaltyClient/VoiceManager.cs

```diff
@@ -489,6 +494,11 @@ internal void SetRadioChannel(string radioChannelName, bool primary)
 
             BaseScript.TriggerServerEvent(Event.SaltyChat_SetRadioChannel, radioChannelName, primary);
         }
+
+        internal void SetRadioVolume(float radioVolumeNewValue)
```

Comment:
You should consider modifying this so there is no possibility to set RadioVolume to an unexpected Value